<launch>
    <param name="use_sim_time" value="true" />
    <node name="stageros" pkg="stage_ros" type="stageros" args="$(find tut_ibx0020)/worlds/test2.world" />

    <group ns = "robot_0">
        <node name="robot_reactive_0" pkg="ibx0020_control" type="reactive.py" output="screen" />
    </group>
    <group ns = "robot_1">
        <node name="robot_reactive_1" pkg="ibx0020_control" type="reactive.py" />
    </group>
    <group ns = "robot_2">
        <node name="robot_reactive_1" pkg="ibx0020_control" type="reactive.py" />
    </group>
    <group ns = "robot_3">
        <node name="robot_reactive_1" pkg="ibx0020_control" type="reactive.py" />
    </group>
    <group ns = "robot_4">
        <node name="robot_reactive_1" pkg="ibx0020_control" type="reactive.py" />
    </group>
    <group ns = "robot_5">
        <node name="robot_reactive_1" pkg="ibx0020_control" type="reactive.py" /> 
    </group>
</launch>